rocAL/rocAL_pybind/examples/rocAL_api_pipeline.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import sys
+import cv2
+import os
+
+def draw_patches(img, idx, device):
+    #image is expected as a tensor, bboxes as numpy
+    import cv2
+    image = img.cpu().numpy()
+    image = image.transpose([1, 2, 0])
```

Comment:
the transpose can be avoided using the correct output format for pipeline

Reply:
done